source/standards/logging.html.md.erb

```diff
@@ -125,13 +124,18 @@ There is [broker documentation] describing how drain logs to Splunk via
 The [GOV.UK PaaS Logging] documentation will help you configure Logit and
 drain logs into it from your app.
 
+###Contact
+
+Any questions regarding storing and querying logs should be directed to 
+Louise Ryan (`louise.ryan@digital.cabinet-office.gov.uk`) in the first instance.
```

Comment:
I feel like this contradicts working in the open - can we not ask @louzoid-gds to monitor #splunk for relevant discussions? :-)

Reply:
Oh I do already galund :)
Kat - can we drop this Contact part for now?